docs/platform/reference/eol-for-major-versions.md

```diff
@@ -93,13 +93,11 @@ after they are made available on the Aiven platform.
 
 ### Aiven for Apache Cassandra® {#h_0f2929c770}
 
-Starting with v4, Aiven for Cassandra® major versions will reach EOL six
-months after they are made available on the Aiven platform.
-
 | Version | Aiven EOL  | Service creation supported until | Service creation supported from |
 | ------- | ---------- | -------------------------------- | ------------------------------- |
 | 3       | 2022-07-27 | 2022-04-27                       | 2018-11-08                      |
-| 4       | N/A        | N/A                              | 2021-12-09                      |
+| 4.0     | 2024-09    | N/A                              | 2021-12-09                      |
```

Comment:
we should use 2024-09-30 and 2025-09-30 - I think we use a well-defined day in all of our products.

Reply:
Now we're discussing a "service creation supported until" date for 4.0 as well.